### PR TITLE
Add bde_ext.ttl_inst_cmplte table

### PR DESCRIPTION
**Pull request by strk**
For discussion

### CODE REVIEW

**Comment by strk** [comment]
PR @75 is another (alternative) option: a view

File: sql/05-bde_ext_schema.sql [hunk 1]
```diff
@@ -878,6 +878,23 @@ REVOKE ALL ON TABLE ttl_inst FROM PUBLIC;
 GRANT SELECT, UPDATE, INSERT, DELETE ON TABLE ttl_inst TO bde_admin;
 GRANT SELECT ON TABLE ttl_inst TO bde_user;
 
+-- =============================================================================
+-- T T L   I N S T   C M P L T E
+-- =============================================================================
+
+DROP TABLE IF EXISTS ttl_inst_cmplte CASCADE;
```
Comment (palmerj) [inline sql/05-bde_ext_schema.sql]: Also changes to the BDE database structure during minor and patch releases need to be applied using the 99-patches.sql file.

Comment (palmerj) [reply sql/05-bde_ext_schema.sql]: Also this table needs to be versioning using the ver_enable_table_versioning() API

Comment (palmerj) [reply sql/05-bde_ext_schema.sql]: Is this approach take to provision a new table endpoint, separate from https://data.linz.govt.nz/table/52013-landonline-title-instrument-title? Is so then this approach of creating a new table is good. If we are just going to update the current LDS table then we should be altering the current ttl_inst table and adding the new column via the `ver_versioned_table_add_column` API in the 99-patches.sql file

Comment (billgeo) [reply sql/05-bde_ext_schema.sql]: @palmerj We need to create a new table.

Comment (palmerj) [reply sql/05-bde_ext_schema.sql]: Great, then we need to create and version the table as a patch

File: sql/06-bde_ext_functions.sql.in [hunk 2]
```diff
@@ -2037,6 +2037,36 @@ BEGIN
...
+    ----------------------------------------------------------------------------
+    v_table := LDS.LDS_GetTable('bde_ext', 'ttl_inst_cmplte');
+
+    v_data_insert_sql := $sql$
+        INSERT INTO %1% (
+                id,
+                completion_date
+        )
+        SELECT
+                TIN.id,
```
Comment (billgeo) [inline sql/06-bde_ext_functions.sql.in]: Can we also insert the other columns from crs_ttl_inst? So ttl_inst_cmplte will have the same columns as ttl_inst but with the additional column of 'completion_date'.

Comment (strk) [reply sql/06-bde_ext_functions.sql.in]: Ok, so full duplication of data. I'm on it.

**Comment by strk** [comment]
As of bd7d7d3  the `bde_ext.ttl_inst_cmplte` table is created with all attributes via patch and versioned when the reference table is also versioned. Please review again. Tests for bde_ext schema composition are also added (were missing before)

**Comment by imincik** [comment]
Thank you @strk

**Pull request merged**